### PR TITLE
Update spec for MVN-1 and MVN-6.

### PR DESCRIPTION
**Pull request by sdurawa**
### Details:
 - update specification for MVN-1 and MVN-6

### Tickets:
 - 45802, 44619

### CODE REVIEW

**Comment by sdurawa** [comment]
@ilyachur, as Evgeny is absent, could you add to this review someone who can help?

**Comment by ilyachur** [comment]
@rkazants Could you have a look?

File: docs/ops/normalization/MVN_1.md [hunk 1]
```diff
@@ -4,58 +4,93 @@
...
+  * **Description**: *across_channels* is a flag that specifies whether mean values are shared across channels. If `true` mean values and variance are calculated for each sample across all channels and spatial dimensions (Layer Normalization), otherwise calculation is done for each sample and for each channel across spatial dimensions (Instance/Contrast Normalization).
   * **Range of values**:
     * `false` - do not share mean values across channels
     * `true` - share mean values across channels
   * **Type**: `boolean`
-  * **Default value**: `false`
+  * **Default value**: None
+  * **Required**: *no*
+
+* *reduction_axes*
```
Comment (sdurawa) [inline docs/ops/normalization/MVN_1.md:50]: I am wondering whether it is a valid attribute for MVN-1? It is implemented in source code as attribute and it is used only for internal test purpose.

Comment (mvafin) [reply docs/ops/normalization/MVN_1.md:50]: It is a valid attribute, because it can be read from IR

File: docs/ops/normalization/MVN_1.md [hunk 2]
```diff
@@ -4,58 +4,93 @@
...
 
   * **Description**: *normalize_variance* is a flag that specifies whether to perform variance normalization.
   * **Range of values**:
-    * `false` -- do not normalize variance
-    * `true` -- normalize variance
+    * `false` - do not normalize variance
+    * `true` - normalize variance
   * **Type**: `boolean`
-  * **Default value**: `false`
+  * **Default value**: None
```
Comment (rkazants) [inline docs/ops/normalization/MVN_1.md:65]: if this is not mandatory attribute, the default value must exist

File: docs/ops/normalization/MVN_1.md [hunk 3]
```diff
@@ -4,58 +4,93 @@
...
   * **Required**: *no*
 
 * *eps*
 
   * **Description**: *eps* is the number to be added to the variance to avoid division by zero when normalizing the value. For example, *epsilon* equal to 0.001 means that 0.001 is added to the variance.
   * **Range of values**: a positive floating-point number
-  * **Type**: `float`
-  * **Default value**: None
+  * **Type**: `double`
+  * **Default value**: 1e-9
```
Comment (rkazants) [inline docs/ops/normalization/MVN_1.md:73]: if this is required attribute, the default value is not needed.

Comment (sdurawa) [reply docs/ops/normalization/MVN_1.md:73]: The reason I updated default value is current MVN-1 implementation:

`private:`
`double m_eps = 1e-9;`
`bool m_across_channels;`
`bool m_normalize_variance;`
`AxisSet m_reduction_axes;`

Shall I change op implementation together with the spec change?

Comment (rkazants) [reply docs/ops/normalization/MVN_1.md:73]: @sdurawa, I think - yes. We need align all implementation with the specification. If `eps` is required, it needs to be checked that this parameter value is passed in nGraph shell, etc.
@mvafin, what do you think?

File: docs/ops/normalization/MVN_1.md (hunk 1 above)
Comment (sdurawa) [reply docs/ops/normalization/MVN_1.md:50]: @mvafin shall we support both attributes?

File: docs/ops/normalization/MVN_1.md (hunk 3 above)
Comment (sdurawa) [reply docs/ops/normalization/MVN_1.md:73]: @mvafin, could you help here?

Comment (mvafin) [reply docs/ops/normalization/MVN_1.md:73]: Agree with @rkazants we need to change implementation to align with spec

File: docs/ops/normalization/MVN_1.md (hunk 1 above)
Comment (mvafin) [reply docs/ops/normalization/MVN_1.md:50]: Yes, it is a legacy op. We don't need to delete any functionality if it was there

File: docs/ops/normalization/MVN_1.md (hunk 3 above)
Comment (sdurawa) [reply docs/ops/normalization/MVN_1.md:73]: @mvafin, I assume the same approach shall be used for MVN-6, right? Current implementation:                
`bool m_normalize_variance = true;`
`float m_eps = (float)1e-6;`
`MVNEpsMode m_eps_mode = MVNEpsMode::INSIDE_SQRT;`

but spec clearly says that attributes have no default values.

File: docs/ops/normalization/MVN_1.md (hunk 2 above)
Comment (sdurawa) [reply docs/ops/normalization/MVN_1.md]: Fixed.

File: docs/ops/normalization/MVN_1.md (hunk 3 above)
Comment (sdurawa) [reply docs/ops/normalization/MVN_1.md]: Fixed.

**Comment by sdurawa** [comment]
@lazarevevgeny , @mvafin , @rkazants all comments are addressed, could you review specification again?

**Review by rkazants — APPROVED** [review]

**Review by mvafin — APPROVED** [review]

**Pull request merged**